polaris/benchmark/_base.py

```diff
@@ -182,30 +182,44 @@ def _validate_split(cls, v, info: ValidationInfo):
             raise InvalidBenchmarkError("The predefined split contains empty test partitions")
 
         train_idx_list = v[0]
-        test_idx_list = list(i for part in v[1].values() for i in part) if isinstance(v[1], dict) else v[1]
+        full_test_idx_list = (
+            list(i for part in v[1].values() for i in part) if isinstance(v[1], dict) else v[1]
+        )
 
         if len(train_idx_list) == 0:
             logger.info(
                 "This benchmark only specifies a test set. It will return an empty train set in `get_train_test_split()`"
             )
 
         train_idx_set = set(train_idx_list)
-        test_idx_set = set(test_idx_list)
+        full_test_idx_set = set(full_test_idx_list)
 
         # The train and test indices do not overlap
-        if len(train_idx_set & test_idx_set) > 0:
+        if len(train_idx_set & full_test_idx_set) > 0:
             raise InvalidBenchmarkError("The predefined split specifies overlapping train and test sets")
 
-        # Duplicate indices
+        # Check for duplicate indices within the train set
         if len(train_idx_set) != len(train_idx_list):
             raise InvalidBenchmarkError("The training set contains duplicate indices")
-        if len(test_idx_set) != len(test_idx_list):
-            raise InvalidBenchmarkError("The test set contains duplicate indices")
+
+        # Check for duplicate indices within a given test set. Because a user can specify
+        # multiple test sets for a given benchmark and it is acceptable for indices to be shared
+        # across test sets, we check for duplicates in each test set independently.
+        if isinstance(v[1], dict):
+            for test_set_name, test_set_idx_list in v[1].items():
+                test_set_idx_set = set(test_set_idx_list)
+                if len(test_set_idx_list) != len(test_set_idx_set):
+                    raise InvalidBenchmarkError(
+                        f'Test set with name "{test_set_name}" contains duplicate indices'
+                    )
+        else:
+            if len(full_test_idx_set) != len(full_test_idx_list):
+                raise InvalidBenchmarkError("The test set contains duplicate indices")
```

Comment:
You can keep it as is if you find the above version more readable:

```suggestion
        elif len(full_test_idx_set) != len(full_test_idx_list):
            raise InvalidBenchmarkError("The test set contains duplicate indices")
```